palace/linalg/errorestimator.cpp

```diff
@@ -63,9 +63,15 @@ ConfigureLinearSolver(const mfem::ParFiniteElementSpaceHierarchy &fespaces, doub
   std::unique_ptr<Solver<Operator>> pc;
   if (fespaces.GetNumLevels() > 1)
   {
+    const int dim = fespaces.GetFinestFESpace().GetParMesh()->Dimension();
+    const auto type = fespaces.GetFinestFESpace().FEColl()->GetRangeType(dim);
+    const int mg_smooth_order =
+        (type == mfem::FiniteElement::SCALAR)
+            ? 2
+            : std::max(fespaces.GetFinestFESpace().GetMaxElementOrder(), 2);
```

Comment:
So the smoothing only needs to be strengthened for ND elements, do we know why?

Reply:
Yes, this is based on experimentation and this seems to be a good heuristic tor optimal settings. A few ideas as to why this could be:
- I'm not sure if this is proven, but I believe the element Nedelec mass matrix is more poorly conditioned than the nodal one. Has to do with continuity, etc. so it would make sense the preconditioner should change.
- In the context of Jacobi-preconditioned polynomial smoothing used for multigrid, the diagonal as assembled in parallel in both the nodal and Nedelec cases is approximate when the operator isn't fully assembled. For Nedelec it introduces a few more approximations and thus makes sense why more iterations for the same smoother order might be required.

Reply:
I've just gone ahead and increased the smoother order for all polynomial smoothing in [b1754fd](https://github.com/awslabs/palace/pull/120/commits/b1754fda2f04a321c5945c4b9d14899456c8ea70).  This is nice for consistency and will keep iteration counts similar at least.